kolibri/utils/server.py

```diff
@@ -322,3 +323,39 @@ def get_urls(listen_port=None):
         return STATUS_RUNNING, urls
     except NotRunning as e:
         return e.status_code, []
+
+
+def installation_type():
+    """
+    Tries to guess how the running kolibri server was installed
+
+    :returns: install_type is the type of detected installation
+    install_type can be any of these strings:
+        - 'PEX file'
+        - 'Debian package'
+        - 'Windows installer'
+        - 'Compiled from code or from a Whl file'
+        - 'Development code'
+        - 'unknown': this function has not been able to guess it
+    """
+    install_type = 'unknown'
+    if len(sys.argv) > 1:
+        launcher = sys.argv[0]
+        if 'start' in sys.argv:
```

Comment:
Anyone can do `kolibri manage runserver` on their installation, thereby bypassing the below conditions.. also, a development installation can run `kolibri start`. I think it's better remove `if 'start' in sys.argv`.

Reply:
if you do `kolibri start`in  a development installation you are not running it from development, but from the 'Compiled code..' . With 'Development code' I want to mean exactly that: running from django-server while coding. Any other kind of call using start will require a compilated version.

Reply:
I have changed the way detection is done for linux.. `start` is now considered only in one case. I think you were right when saying that start could be used after a Deb package was installed. The latest changes should solve that ambiguity.